### DISCUSSION TITLE
Status of cryptopp

### DISCUSSION THREAD

**Discussion by trg818** [Homebrew/discussions#1172] [Everyday usage]
In my effort to compile Clementine for Mac Silicon, I find that Homebrew does not have the package `cryptopp` which according [these instructions](https://github.com/clementine-player/Clementine/wiki/Compiling-from-source-(macOS)) is needed. It seems that some error in connection with `cryptopp` is somehow on record at Homebrew, but I didn't find any further information. Does anybody know about the status of the problem or have suggestions on how to resolve it?

**Comment by SMillerDev** [comment]
See https://github.com/Homebrew/homebrew-core/issues/56394, it doesn't work on modern macOS versions in a way that works with homebrew so it was removed.

**Comment by carlocab** [comment]
You should be able to `brew extract` it to your own tap and build it from there, as it currently isn't suited for distribution via homebrew/core.

Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

You may also wish to do `brew help extract` and `brew help tap-new`.
